src/pages/Teams/ListTeams/ListTeams.jsx

```diff
@@ -0,0 +1,124 @@
+import React, { useEffect } from 'react';
+import {
+  Button,
+  Stack,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Tooltip,
+  Typography,
+} from '@mui/material';
+import GroupAddIcon from '@mui/icons-material/GroupAdd';
+import PersonAddIcon from '@mui/icons-material/PersonAdd';
+import EditIcon from '@mui/icons-material/Edit';
+import DeleteIcon from '@mui/icons-material/Delete';
+import { useNavigate } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { getTeams } from '../../../store/actions/teams';
+
+function capitalize(text) {
+  return text.charAt(0).toUpperCase() + text.slice(1);
+}
+
+const ListTeams = () => {
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const { teams } = useSelector((state) => state.teams);
+
+  useEffect(() => {
+    dispatch(getTeams());
+  }, [dispatch]);
+
+  return (
+    <>
+      <Stack direction="row" justifyContent="space-between" alignItems="flex-start" spacing={2}>
+        <Typography variant="h3" component="h3">
+          List of Teams
+        </Typography>
+        <Button
+          variant="outlined"
+          onClick={() => {
+            navigate('/teams/add');
+          }}
+        >
+          <GroupAddIcon
+            sx={{
+              padding: '0 10px 0 0',
+            }}
+          />
+          Add a team
+        </Button>
+      </Stack>
+      <TableContainer>
+        <Table>
+          <TableHead>
+            <TableRow>
+              <TableCell>Name</TableCell>
+              <TableCell align="center">Projects</TableCell>
+              <TableCell align="center">Users</TableCell>
+              <TableCell />
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {teams.map((team) => (
+              <TableRow key={team._id}>
+                <TableCell>{team.name}</TableCell>
+                <TableCell align="center">{team.projects.length}</TableCell>
+                <Tooltip
+                  placement="right"
+                  title={
+                    <div>
+                      {team.users.map((user) => (
+                        <div key={user._id}>
+                          {capitalize(user.user.firstName)}
+                          {user.role === 'admin' ? ' (Admin)' : ''}
+                        </div>
+                      ))}
+                    </div>
+                  }
+                >
```

Comment:
Could you add a condition around this, if length of users in the team is equal to 0 to avoid this render :

<img width="213" alt="Screenshot 2022-12-21 at 18 59 37" src="https://user-images.githubusercontent.com/66887387/208972928-6241ad3a-361b-40c5-921c-f4ccc53c82ce.png">